README.md

```diff
@@ -147,7 +147,7 @@ env MONGO_HOST=127.0.0.1 TAP_BAIL=1 node tests/createIndexes.test.js
 +----------------------------------------------------------------------------------------------------+
 ```
 
-The CRUD Service application uses Fastify to connect to a MongoDB instance (thanks to the @fastify/mongodb plugin) and execute CRUD operations. At startup, it requires several informations, such as the mongoDB URL to communicate with, but also the folders where there are included the definitions of the collections and the views. A complete and comprehensive list of the environment variables needed can be found in the [Configuration page](./docs/20_Configuration.md).
+The CRUD Service application uses Fastify to connect to a MongoDB instance (thanks to the @fastify/mongodb plugin) and execute CRUD operations. At startup, it requires several details, such as the MongoDB URL to communicate with, but also the folders where the definitions of the collections and the views can be found. A complete and comprehensive list of the environment variables needed can be found in the [Configuration page](./docs/20_Configuration.md).
```

Comment:
Since we're here, I'll add some grammar fix.

```suggestion
The CRUD Service application uses Fastify to connect to a MongoDB instance (thanks to the @fastify/mongodb plugin) and execute CRUD operations. At startup, it requires several details, such as the MongoDB URL to communicate with, but also the folders where the definitions of the collections and the views can be found. A complete and comprehensive list of the environment variables needed can be found on the [Configuration page](./docs/20_Configuration.md).
```